spring-integration-core/src/main/java/org/springframework/integration/scheduling/SimplePollSkipStrategy.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.scheduling;
+
+/**
+ * A simple {@link PollSkipStrategy} to be used with a {@code PollSkipAdvice}.
+ * Invoke {@link #skipPolls()} to start skipping polls; invoke {@link #reset()}
+ * to resume polling.
+ *
+ * @author Gary Russell
+ * @since 4.2.5
+ *
+ */
+public class SimplePollSkipStrategy implements PollSkipStrategy {
```

Comment:
This is really cool simple feature!
I wonder how we have missed it before... :smile: 

Anyway it looks like a `@ManagedResource` candidate to be able to change the state using `Control Bus`.